### PR TITLE
Added UG document for No Result found template

### PR DESCRIPTION
**Pull request by YuvarajMasilamani**
Hi @SubashiniMahendran 

I have updated a UG document for the No Result Found Template in AutoComplete. Please review and let me know if you have any concerns.

Regards
Yuvaraj M

### CODE REVIEW

File: winui/AutoComplete/No-Result-Found.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+title: No Result Found in WinUI AutoComplete control | Syncfusion
+description: Learn how to use NoResultsFoundTemplate and NoResultsFoundContent in Syncfusion WinUI AutoComplete control.
+platform: winui
+control: SfAutoComplete
+documentation: ug
+---
+
+# No Result Found  in WinUI AutoComplete (SfAutoComplete)
+
+When the entered item is not in the suggestion list, SfAutoComplete displays a text indicating there is no search results found. We can set the desire text to be displayed for indicating no results found with the [NoResultsFoundContent] property and  the desire text to be displayed for indicating no results found can be customized using the [NoResultsFoundTemplate] property.
```
Comment (SubashiniMahendran) [inline winui/AutoComplete/No-Result-Found.md:12]: @YuvarajMasilamani, why you have used [ for properties. Remove this and use highlight symbol after publishing in live we can provide hyperlink for API

File: winui/AutoComplete/No-Result-Found.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+documentation: ug
+---
+
+# No Result Found  in WinUI AutoComplete (SfAutoComplete)
+
+When the entered item is not in the suggestion list, SfAutoComplete displays a text indicating there is no search results found. We can set the desire text to be displayed for indicating no results found with the [NoResultsFoundContent] property and  the desire text to be displayed for indicating no results found can be customized using the [NoResultsFoundTemplate] property. 
+
+## NoResultsFoundContent 
+
+The below code shows how to include the `NoResultsFoundContent` in AutoComplete.
```
Comment (SubashiniMahendran) [inline winui/AutoComplete/No-Result-Found.md:16]: @YuvarajMasilamani, include more details in this topic about the use case of the properties, example, screenshot. Compare competitor and our other platform.

File: winui/AutoComplete/No-Result-Found.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+ </editors:SfAutoComplete>
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AutoComplete control with NoResultsFoundContent](No_Results_Found_images\No-Results-Found-Content.png)
+
+## NoResultsFoundTemplate 
+
+The below code shows how to include the `NoResultsFoundTemplate` in AutoComplete.
```
Comment (SubashiniMahendran) [inline winui/AutoComplete/No-Result-Found.md:37]: @YuvarajMasilamani, include more details in this topic about the use case of the properties, example, screenshot. Compare competitor and our other platform.

File: winui/AutoComplete/No-Result-Found.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                </DataTemplate>
+            </editors:SfAutoComplete.NoResultsFoundTemplate>
+ </editors:SfAutoComplete>
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AutoComplete control with NoResultsFoundTemplate](No_Results_Found_images\No-Results-Found-Template.png)
+
+## NoResultsFoundContent and NoResultsFoundTemplate
```
Comment (SubashiniMahendran) [inline winui/AutoComplete/No-Result-Found.md:63]: @YuvarajMasilamani, remove this topic and in notes we can mention it as while setting both properties template property have higher preference.

**Review by SubashiniMahendran — CHANGES_REQUESTED** [review]

File: winui/AutoComplete/No-Result-Found.md (hunk 1 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/No-Result-Found.md]: Modified

File: winui/AutoComplete/No-Result-Found.md (hunk 3 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/No-Result-Found.md]: Updated

File: winui/AutoComplete/No-Result-Found.md (hunk 4 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/No-Result-Found.md]: resolved

File: winui/AutoComplete/No-Result-Found.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: No Result Found in WinUI AutoComplete control | Syncfusion
+description: Learn how to use NoResultsFoundTemplate and NoResultsFoundContent in Syncfusion WinUI AutoComplete control.
+platform: winui
+control: SfAutoComplete
+documentation: ug
+---
+
+# No Result Found  in WinUI AutoComplete (SfAutoComplete)
```
Comment (SubashiniMahendran) [inline winui/AutoComplete/No-Result-Found.md:10]: @YuvarajMasilamani, remove extra space in between found and in

File: winui/AutoComplete/No-Result-Found.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+title: No Result Found in WinUI AutoComplete control | Syncfusion
+description: Learn how to use NoResultsFoundTemplate and NoResultsFoundContent in Syncfusion WinUI AutoComplete control.
+platform: winui
+control: SfAutoComplete
+documentation: ug
+---
+
+# No Result Found  in WinUI AutoComplete (SfAutoComplete)
+
+When the entered item is not in the suggestion list, SfAutoComplete displays a text indicating there is no search results found. We can set the desire text to be displayed for indicating no results found with the `NoResultsFoundContent` property and  the desire text to be displayed for indicating no results found can be customized using the `NoResultsFoundTemplate` property.
```
Comment (SubashiniMahendran) [inline winui/AutoComplete/No-Result-Found.md:12]: @YuvarajMasilamani, change this sentence as

When the entered item is not in the suggestion list, SfAutoComplete displays a text indicating there is no search results found. We can set the desire text to be displayed for indicating no results found with the `NoResultsFoundContent` and `NoResultsFoundTemplate` properties.

File: winui/AutoComplete/No-Result-Found.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+documentation: ug
+---
+
+# No Result Found  in WinUI AutoComplete (SfAutoComplete)
+
+When the entered item is not in the suggestion list, SfAutoComplete displays a text indicating there is no search results found. We can set the desire text to be displayed for indicating no results found with the `NoResultsFoundContent` property and  the desire text to be displayed for indicating no results found can be customized using the `NoResultsFoundTemplate` property. 
+
+## NoResultsFoundContent 
+
+We can customize the Displaying text by using the `NoResultsFoundContent` property. The below code shows how to include the `NoResultsFoundContent` in AutoComplete.
```
Comment (SubashiniMahendran) [inline winui/AutoComplete/No-Result-Found.md:16]: @YuvarajMasilamani, change sentence as 

We can customize the desire text to be displayed for indicating no results found by using the `NoResultsFoundContent` property. The below code shows how to include the `NoResultsFoundContent` in AutoComplete.

File: winui/AutoComplete/No-Result-Found.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+ </editors:SfAutoComplete>
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AutoComplete control with NoResultsFoundContent](No_Results_Found_images\No-Results-Found-Content.png)
+
+## NoResultsFoundTemplate 
+
+We can customize the appearance of the displaying text by using the `NoResultsFoundTemplate` property. The below code shows how to include the `NoResultsFoundTemplate` in AutoComplete.
```
Comment (SubashiniMahendran) [inline winui/AutoComplete/No-Result-Found.md:37]: @YuvarajMasilamani, change content as

We can customize the appearance of the desire text to be displayed for indicating no results found by using the `NoResultsFoundTemplate` property. The below code shows how to include the `NoResultsFoundTemplate` in AutoComplete.

File: winui/AutoComplete/No-Result-Found.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+                </DataTemplate>
+            </editors:SfAutoComplete.NoResultsFoundTemplate>
+ </editors:SfAutoComplete>
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AutoComplete control with NoResultsFoundTemplate](No_Results_Found_images\No-Results-Found-Template.png)
+
+N> While setting `NoResultsFoundContent` and `NoResultsFoundTemplate` properties. High priority is given to`NoResultsFoundTemplate`.
```
Comment (SubashiniMahendran) [inline winui/AutoComplete/No-Result-Found.md:63]: @YuvarajMasilamani, provide proper spacing in this sentence for last word.

File: winui/AutoComplete/No-Result-Found.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
+---
```
Comment (SubashiniMahendran) [inline winui/AutoComplete/No-Result-Found.md:1]: @YuvarajMasilamani, add this topic in winui toc.html file under autocomplete topic after searching and filtering.

**Review by SubashiniMahendran — CHANGES_REQUESTED** [review]

File: winui/AutoComplete/No-Result-Found.md (hunk 2 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/No-Result-Found.md]: resolved

File: winui/AutoComplete/No-Result-Found.md (hunk 10 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/No-Result-Found.md:1]: updated

File: winui/AutoComplete/No-Result-Found.md (hunk 5 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/No-Result-Found.md]: resolved

File: winui/AutoComplete/No-Result-Found.md (hunk 6 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/No-Result-Found.md]: Updated

File: winui/AutoComplete/No-Result-Found.md (hunk 8 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/No-Result-Found.md]: Updated

File: winui/AutoComplete/No-Result-Found.md (hunk 9 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/No-Result-Found.md]: Resolved

File: winui/AutoComplete/No-Result-Found.md (hunk 7 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/No-Result-Found.md]: Updated

File: winui/AutoComplete/No-Result-Found.md (hunk 10 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/No-Result-Found.md:1]: Updated

File: winui-toc.html [hunk 11]
```diff
@@ -78,6 +78,7 @@
             <li><a href="/winui/AutoComplete/getting-started">Getting Started</a></li>
             <li><a href="/winui/AutoComplete/selection">Selection</a></li>
             <li><a href="/winui/AutoComplete/searching-filtering">Searching and Filtering</a></li>
+            <li><a href="/winui/AutoComplete/no-result-found-template">No Result Found Template</a></li>
             <li><a href="/winui/AutoComplete/ui-customization">UI Customization</a></li>
```
Comment (SubashiniMahendran) [inline winui-toc.html:82]: @YuvarajMasilamani, you need to mention md file name in href

<li><a href="/winui/AutoComplete/no-result-found">No Result Found</a></li>

**Review by SubashiniMahendran — CHANGES_REQUESTED** [review]

**Review by SubashiniMahendran — APPROVED** [review]

**Pull request merged**